src/Package/MSTest.TestAdapter.symbols.nuspec

```diff
@@ -42,50 +42,63 @@
   <files>
     <!-- netstandard2.0 -->
     <file src="Build\NetCore\MSTest.TestAdapter.props" target="build\netstandard2.0\" />
+    <file src="MSTest.Core\netstandard2.0\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\netstandard2.0\" />
     <file src="PlatformServices\netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.dll" target="build\netstandard2.0\" />
+    <file src="PlatformServices\netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.dll" target="build\netstandard2.0\" />
     <file src="Microsoft.TestPlatform.AdapterUtilities\netstandard2.0\" target="build\netstandard2.0\" />
     <file src="MSTest.TestAdapter\netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter.dll" target="build\netstandard2.0\" />
-    <file src="PlatformServices.Interface\netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.dll" target="build\netstandard2.0\" />
-    <file src="MSTest.Core\netstandard2.0\Microsoft.VisualStudio.TestPlatform.TestFramework.dll" target="build\netstandard2.0\" />
-
     <!-- Symbols -->
     <file src="PlatformServices\netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.pdb" target="build\netstandard2.0\" />
+    <file src="PlatformServices\netstandard2.0\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.pdb" target="build\netstandard2.0\" />
```

Comment:
Is it normal that we don't include `Microsoft.TestPlatform.AdapterUtilities` and `MSTest.TestAdapter` pdbs? This is true for all TFMs.

Reply:
Yes. Those are cached in our symbols servers.